### DISCUSSION TITLE
How do I set the log file path?

### DISCUSSION THREAD

**Discussion by crenelle** [CocoaLumberjack/CocoaLumberjack#1366] [Q&A] [answered]
I spotted logsDirectory, but it is read-only.

**Comment by ffried** [comment, marked as answer]
You need to create a log file manager for this. You can use the [`DDLogFileManagerDefault`'s initializer](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/Sources/CocoaLumberjack/include/CocoaLumberjack/DDFileLogger.h#L211) to pass a `logsDirectory`.

**Reply by crenelle** [reply]
Tnx!

-Mike
